protocol/avdl/chat1/common.avdl

```diff
@@ -593,7 +600,10 @@
 
   record ChatSearchInboxHit {
     ConversationID convID;
+    TeamType teamType;
     string convName;
+    string query;
```

Comment:
this info is repeated on all chat hits from the search call, can it be threaded threaded in a different way?

Reply:
No, I did this for convenience on the UI side.